File: src/main/scala/work/chiro/game/bullet/EnemyBullet.scala
package work.chiro.game.bullet

import work.chiro.game.animate.AnimateContainer
import work.chiro.game.application.ImageResourceFactory
import work.chiro.game.basic.ScaleType.Position

/**
 * @author chiro2001
 */
class EnemyBullet(posInit: Position, animateContainer: AnimateContainer, power: Int)
  extends AbstractBullet(posInit, animateContainer, power) {
  override def getImage = EnemyBullet.getImage
}

object EnemyBullet extends ImageResourceFactory {
  override def getImageCachedPath = "images/bullet_enemy.png"
}

File: core/src/main/java/work/chiro/game/objects/bullet/EnemyBullet.java
package work.chiro.game.objects.bullet;

import work.chiro.game.animate.AnimateContainer;
import work.chiro.game.vector.Vec2;

/**
 * @author hitsz
 */
public class EnemyBullet extends BaseBullet {
    public EnemyBullet(Vec2 posInit, AnimateContainer animateContainer, int power) {
        super(posInit, animateContainer, power);
    }
}
